test/quantization/algorithm/test_gptq.py

```diff
@@ -97,6 +101,8 @@ def test_net(self):
         q_m = convert(q_m)
 
         # Export circle
+        # pt2e exported model doesn't have `eval()` api.
+        q_m.training = False
         cm = tico.convert(q_m, example_inputs)
         cm.save("op.q.circle")
```

Comment:
This is not about this change but, how bout saving this into temporal directory?

Reply:
On second thought, we don't need to save the circle file. I'll remove this.